src/improvements/SuperchainAddressRegistry.sol

```diff
@@ -151,25 +172,38 @@ contract SuperchainAddressRegistry is StdChains {
     }
 
     /// @notice Retrieves an address by its identifier for a specified L2 chain
+    /// This is deprecated in favor of the `get` function.
```

Comment:
We know it returns an address from the return type, and since we'll use `registry.get(...)` a lot in tasks I figured we should just shorten the interface here. Similar to how we only have `get` in the simple address registry also